ui/pages/confirmations/components/signature-request-original/signature-request-original.test.js

```diff
@@ -191,29 +190,4 @@ describe('SignatureRequestOriginal', () => {
     render();
     expect(screen.getByText('This is a deceptive request')).toBeInTheDocument();
   });
-
-  it('should display mismatch info when selected account address and from account address are not the same', () => {
```

Comment:
Not clear how this is related to MMI

Reply:
Oh nevermind, I see that this warning was exclusive to MMI builds, makes sense. I was fooled by the test not using a custodial account.